File: tema-02/operacionesMatematicas.php
<!DOCTYPE html>
<!--
Escribe un programa que sume, reste, multiplique y divida dos números introducidos por teclado.
-->
<html>
    <head>
        <title>Operaciones Matematicas</title>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
    </head>
    <body>
        <div>
            <h3>Operaciones Matematicas</h3>
            <?php
                $x = $_GET["x"];
                $y = $_GET["y"];
                $suma = $x + $y;
                $resta = $x - $y;
                $multiplica = $x * $y;
                $divide = $x / $y;
            ?>
            
            <table style="border-color: black 1px; border-style: groove">
                <tr><td>Suma</td><td>Resta</td><td>Multiplicación</td><td>División</td></tr>
                <tr><td><?php echo $suma ?></td><td><?php echo $resta ?></td><td><?php echo $multiplica ?></td><td><?php echo $divide ?></td></tr>
            </table>
        </div>
    </body>
</html>

File: capitolul3/src/capitolul3/OperacionesMatematicas.java
/*
Escribe un programa que sume, reste, multiplique y divida dos números
introducidos por teclado.
 */
package capitolul3;

import java.util.Scanner;

/**
 *
 * @author costy
 */
public class OperacionesMatematicas {
    public static void main(String[] args) {
        Scanner t = new Scanner(System.in);
        
        double suma;
        double resta;
        double multiplica;
        double restar;
        
        int numUno;
        int numDos;
        
        System.out.println("Introduce dos números enteros:");
        numUno = t.nextInt();
        numDos = t.nextInt();
        
        suma = (double)(numUno + numDos);
        resta = (double)(numUno - numDos);
        multiplica = (double)(numUno * numDos);
        restar = (double)(numUno / numDos);
        
        System.out.println(".........................................");
        System.out.println("Los resultados son: ");
        System.out.print("Suma " + suma + "\nResta " + resta + 
                "\nMultiplicación " + multiplica + "\nRestar " + restar);
    }
}
